docs/matrix.md

```diff
@@ -38,17 +41,19 @@ Documentation for specifying `Matrix` in a `Pipeline`:
 ## Configuring a Matrix
 
 A `Matrix` supports the following features:
-* [Concurrency Control](#concurrency-control)
-* [Parameters](#parameters)
-* [Context Variables](#context-variables)
-* [Results](#results)
+
+- [Concurrency Control](#concurrency-control)
+- [Parameters](#parameters)
+- [Context Variables](#context-variables)
+- [Results](#results)
 
 ### Concurrency Control
 
 The default maximum count of `TaskRuns` or `Runs` from a given `Matrix` is **256**. To customize the maximum count of
 `TaskRuns` or `Runs` generated from a given `Matrix`, configure the `default-max-matrix-combinations-count` in
 [config defaults](/config/config-defaults.yaml). When a `Matrix` in `PipelineTask` would generate more than the maximum
-`TaskRuns` or `Runs`, the `Pipeline` validation would fail.
+`TaskRuns` or `Runs`, the `Pipeline` validation would fail. Note: The matrix combination count includes combinations generated
+from `Matrix.Params` and `Matrix.Include` Params.
```

Comment:
```suggestion
`TaskRuns` or `Runs`, the `Pipeline` validation would fail. 

Note: The matrix combination count includes combinations generated from both `Matrix.Params` and `Matrix.Include.Params`.
```